CSharp_Course_Project/Task2_PrinterError.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharp_Course_Project
+{
+    class Task2_PrinterError
+    {
+        public static void PrinterError()
+        {
+            string controlString = GetControlString();
```

Comment:
When you invoke a static method, please use name of class.
Please follow that rule.